base/compiler/ssair/passes.jl

```diff
@@ -736,14 +756,13 @@ function getfield_elim_pass!(ir::IRCode)
             @assert val !== nothing
         end
 
-        global assertion_counter
-        assertion_counter::Int += 1
+        # global assertion_counter
+        # assertion_counter::Int += 1
```

Comment:
🙀

Reply:
We need another optimizer that optimizes our optimizer :p

Reply:
A bit seriously, we may want to put more efforts on integrating JET with Julia base.